Documentation/network/bgp-control-plane/bgp-control-plane.rst

```diff
@@ -18,7 +18,8 @@ reachability within the cluster.
 .. admonition:: Video
   :class: attention
 
-  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
+  For more insights on Cilium's BGP, check out `eCHO episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
+  
```

Comment:
Git says you have trailing spaces on this line that you had, please let's not introduce new trailing spaces if possible :)

Reply:
fixed